modules/rubiconBidAdapter.js

```diff
@@ -843,27 +843,29 @@ export function determineRubiconVideoSizeId(bid) {
   return utils.deepAccess(bid, `mediaTypes.${VIDEO}.context`) === 'outstream' ? 203 : 201;
 }
 
+/**
+ * @param {PrebidConfig} config
+ * @returns {{ranges: {ranges: Object[]}}}
+ */
 export function getPriceGranularity(config) {
-  const granularityMappings = {
-    low: [{max: 5.00, increment: 0.50}],
-    medium: [{max: 20.00, increment: 0.10}],
-    high: [{max: 20.00, increment: 0.01}],
-    auto: [
-      {max: 5.00, increment: 0.05},
-      {min: 5.00, max: 10.00, increment: 0.10},
-      {min: 10.00, max: 20.00, increment: 0.50}
-    ],
-    dense: [
-      {max: 3.00, increment: 0.01},
-      {min: 3.00, max: 8.00, increment: 0.05},
-      {min: 8.00, max: 20.00, increment: 0.50}
-    ]
-  }
-  if (config.getConfig('priceGranularity') === 'custom') {
-    return {ranges: config.getConfig('customPriceGranularity').buckets}
-  } else {
-    return {ranges: granularityMappings[config.getConfig('priceGranularity')]}
-  }
+  return {
+    ranges: {
+      low: [{max: 5.00, increment: 0.50}],
+      medium: [{max: 20.00, increment: 0.10}],
+      high: [{max: 20.00, increment: 0.01}],
+      auto: [
+        {max: 5.00, increment: 0.05},
+        {min: 5.00, max: 10.00, increment: 0.10},
+        {min: 10.00, max: 20.00, increment: 0.50}
+      ],
+      dense: [
+        {max: 3.00, increment: 0.01},
+        {min: 3.00, max: 8.00, increment: 0.05},
+        {min: 8.00, max: 20.00, increment: 0.50}
+      ],
+      custom: config.getConfig('priceGranularity') === 'custom' ? config.getConfig('customPriceBucket').buckets : undefined
```

Comment:
where does `customPriceBucket` come from?  and when does `priceGranularity === 'custom`?  I don't see either in the [Prebid documentation](http://prebid.org/dev-docs/publisher-api-reference.html#setConfig-Price-Granularity).  Is this something custom?

Reply:
no pun intended with my use of "custom"!

Reply:
in a `setConfig` call like so:
```javascript
pbjs.setConfig({
                priceGranularity:{buckets:[{min:.01,max:3,increment:.01},{min:3,max:8,increment:.05},{min:8,max:20,increment:.5},{min:20,max:35,increment:1}]}
            });
```

instead of

```javascript
pbjs.setConfig({
                priceGranularity: "high" || "low" .. etc
            });
```

Reply:
@robertrmartinez showed me that PBJS core changes the `priceGranularity` config to `custom` and puts the buckets in `customPriceBucket`.